### PR TITLE
Update Travis-CI configuration

### PR DESCRIPTION
**Pull request by Arcanemagus**
Includes:

* Building on both stable and beta channels of Atom.
* Proper execution of the build script

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,28 @@
+### Project specific config ###
 language: objective-c
 
+env:
+  matrix:
+    - ATOM_CHANNEL=stable
+    - ATOM_CHANNEL=beta
+
+install:
+  - brew update && brew install swiftlint
```
Comment (norio-nomura) [inline .travis.yml]: swiftlint is pre-installed on Travis. So we should use:
```yaml
before_install:
  - brew update
  - brew outdated swiftlint || brew upgrade swiftlint
```
https://docs.travis-ci.com/user/osx-ci-environment/#A-note-on-upgrading-packages

**Comment by norio-nomura** [comment]
Default image of `language: objective-c` is macOS 10.11 with Xcode 7.3.1.
We should add `osx_image: xcode8.2` for avoiding error on #44.

**Comment by Arcanemagus** [comment]
Pushed up your suggested changes, let's see how this goes, thanks!

**Comment by norio-nomura** [comment]
We have also changed the runtime requirement of SwiftLint in Homebrew distribution.
https://github.com/Homebrew/homebrew-core/pull/9989
Now, SwiftLint can be installed by using Homebrew binary package with Xcode 7.0 or later.

**Comment by Arcanemagus** [comment]
Good to know, so at some point [this line](https://github.com/AtomLinter/linter-swiftlint/blob/54c75eb43d523f5700351a7257110dce210dc2cf/.travis.yml#L3) could be removed.

**Pull request merged**